posthog/models/event/query_event_list.py

```diff
@@ -160,12 +160,22 @@ def run_events_query(
     person_id = query.personId
     order_by = query.orderBy
     select = query.select
-    where = query.where
+    where = []  # make a new object since we'll be modifying it
+    if query.where:
+        where.extend(query.where)
```

Comment:
_Very_ small optimization
```suggestion
    where = query.where.copy() if query.where else [] # Shallow-copy since we'll be modifying it
```